pyet/radiation.py

```diff
@@ -104,8 +104,11 @@ def jensen_haise(tmean, rs=None, cr=0.025, tx=-3, lat=None, method=0, clip_zero=
         if lat is None:
             raise Exception("If you choose method == 1, provide lat!")
         index = get_index(tmean)
-        ra = extraterrestrial_r(index, lat, tmean)
-        pet = ra * (tmean + 5) / 68 / lambd
+        ra = extraterrestrial_r(index, lat)
+        
+        temp = (tmean + 5) / 68 / lambd
```

Comment:
Can i suggest to put the parenthesis here to make clear which division is made first.

I'm sure the code is correct but it is not clear which is the first division :)